File: tada/src/main/java/com/qadri/tada/dto/TaskDto.java
package com.qadri.tada.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.time.Instant;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class TaskDto {
    private Long id;
    private String taskName;
    private String taskDescription;
    private Boolean isCompleted;
    private Boolean isDeleted;
    private Instant lastUpdateTime;
    private Instant createdAt;
}

File: android-app/app/src/main/java/com/qadri/to_do/model/dto/TaskDto.kt
package com.qadri.to_do.model.dto

data class TaskDto(
    val id: Long,
    val taskName: String,
    val taskDescription: String,
    val isCompleted: Boolean,
    val isDeleted: Boolean,
    val lastUpdateTime: Long,
    val createdAt: Long
)
